diffpy/pdfmorph/pdfplot.py

```diff
@@ -207,6 +208,75 @@ def comparePDFs(
     return
 
 
+def plot_rws(target_labels, rws, field=None):
+    """
+    Plot Rw values for multiple morphs.
+    :param target_labels: Names (or field if --sort-by given) of each file acting as target for the morph.
+    :type target_labels: list
+    :param rws: Contains the Rw values corresponding to each file.
+    :type rws: list
+    :param field: When not None and entries in field are numerical, a line chart of Rw versus field is made.
+                  When None (default) or values are non-numerical, it plots a bar chart of Rw values per file.
+    :type field: list or None
+    """
+
+    # ensure all entries in target_labels are distinct for plotting
+    unique_labels = set()
```

Comment:
Could happen if the field given to `--sort-by` is duplicated across multiple files. We handle this by renaming the duplicated field with an additional (#).

![image](https://github.com/diffpy/diffpy.pdfmorph/assets/59151395/0ec0fa71-0b81-4d67-8150-ecb10a33ba74)